### PR TITLE
fix: Validate debug session SSN server side

### PR DESCRIPTION
**Pull request by Youakeem**
## Describe your changes
Remove the fallback test SSN and validate input on the server side.

## Justify why they are needed
We agreed to not fallback to a test SSN for this feature and handle missing input on the server.

## Checklist before requesting a review

- [x] I have performed a self-review of my code

### CODE REVIEW

**Comment by Youakeem** [comment]
* **#4074** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4074?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @Youakeem and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/app/debugger/actions.ts [hunk 1]
```diff
@@ -21,20 +21,34 @@ import { PageLink } from '@/utils/PageLink'
...
 const getRandomEmailAddress = () => {
   const randomId = Math.random().toString(36).substring(2, 5)
   return `sven.svensson.${randomId}@hedvig.com`
 }
 
-export const createCustomerSession = async (formData: FormData) => {
-  try {
-    const ssn = formData.get('ssn') || TEST_SSN
+type CreateCustomerSessionFormState = null | {
+  errors?: {
```
Comment (alebedev) [inline apps/store/src/app/debugger/actions.ts:32]: Let's think how we can model this better
- Most errors would be non-field specific.  We can probably assume at most one such error at a time
- do we want some explicit `isSuccess: true` or something similar?  Relying on absence of errors is less obvious

Comment (Youakeem) [reply apps/store/src/app/debugger/actions.ts:32]: We could provide an `isSuccess` flag in the response. However, with server actions, we don't usually expect a response.

A server action can either:
1. Succeed: Usually revalidate or redirect
2. Throws: Show the error component
3. Return specific errors: Show them to the user and, in this case, we'll intentionally access the errors object anyways.

We could have a `generic` errors as well

```
errors?: {
  fields: { ... }
  generic: string[]
}
```

**Review by alebedev — APPROVED** [review]

**Review by gustaveen — APPROVED** [review]
Looks good!

**Pull request merged**